draft-ietf-capport-architecture.xml

```diff
@@ -695,7 +695,27 @@ o . . . . . . . . . . . . . . . . . . .| |. . . . . . . . . . . o
                 link. In such cases, identification could be performed by subnet, such as the /64
                 to which the IP belongs.
             </t>
-           </section>
+         </section>
+         <section anchor="context_free_url" title="Context-free URL">
+           <t>
+             The URLs provided in the API SHOULD contain all the information necessary to render
+             the resources requested: they should be context free so it does not depend on ambient
+             information, such as remote address on the connection. This is to ensure that if
+             User Equipment access a URL from another network path, the correct resource can be
+             retrieved. One possible approach is to include the identification of the User
+             Equipment in the URLs.
+           </t>
+           <t>
+             The URLs provided in the API are recommended to be resolved using public DNS, so they
+             can be rendered in case User Equipment is making request from a different network.
+           </t>
+           <t>
+             Having User Equipment identification in the URL may expose the service to spoofing or
```

Comment:
Maybe you can also say "Though a URL might still correctly resolve when the UE makes the request from a different network, it is possible that some functions could be limited to when the UE makes requests using the captive network."

Reply:
Do we also need to have some text (possibly in another section) about the wisdom of having UE identifying information in the URL, especially if these URLs might be accessed from off-captive-net?  (one option is to include an authenticated hash of relevant bits or a just unique session that only the captive portal infrastructure can convert into UE-identifying information.)

Reply:
added this, thanks